src/Microsoft.AspNetCore.Blazor/Routing/Router.cs

```diff
@@ -78,11 +88,24 @@ private void Refresh()
         {
             var locationPath = UriHelper.ToBaseRelativePath(_baseUriPrefix, _locationAbsolute);
             locationPath = StringUntilAny(locationPath, _queryOrHashStartChar);
-            var context = new RouteContext(locationPath);
+            var resolvedComponent = GetComponentForPath(locationPath);
+
+            _renderHandle.Render(builder => Render(builder, resolvedComponent.Handler, resolvedComponent.ComponentParameters));
+        }
+
+        /// <summary>
+        /// Decides what <see cref="IComponent"/> should handle a given <paramref name="path"/>
+        /// an the parameters associated with it.
+        /// </summary>
```

Comment:
Typos: Decides **which** `<see cref="IComponent"/>` should handle a given <paramref name="path"/> **and** the parameters associated with it.